_component_design/choice-tiles.md

```diff
@@ -0,0 +1,224 @@
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Choice tiles on a full-width grid with no right sidebar"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-layout.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+### Responsive design specifications
+<div class="hxRow">
+{% column left:"hxCol-10 hxCol-xs-12 hxCol-sm-12 hxCol-md-10 hxCol-lg-10" %}
+
+Below are the column widths for the small, medium, and large choice tiles. Below each, you’ll see a “rowmax” value. This is to indicate our guidance on how many tiles should go in any given row on the corresponding screen size. We limit the amount of tiles in a row to reduce the complexity of the interface for the user, and give them chunks of information they can understand and work with. For pages with a left nav and/or a right sidebar, the 12 column grid spans the container between the sidebar.
+
+{% endcolumn %}
+
+{% column right:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+{% figure [caption:"Choice tiles responsive options"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-responsive.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Choice tile specifications
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Small size choice tiles **cannot** have a description due to the smaller dimensions.
+- A description medium and large size choice tiles is optional, but recommended for supporting more complex selections.
+- When a description is not used, collapse the padding down to the bottom most padding.
+- Choice tiles do not define specific iconography. Stay within brand guidelines and use iconography that supports the selection contained in a choice tile.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Small, medium and large sized choice tiles"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-specs.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+### Basic states
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Choice tiles take on most of the same styling as we use for components like radios and checkboxes, with the addition of the checkmark to indicate a selected tile. The user can click on any part of the tile to select it.
+
+For selected states, the iconography changes color to match the given state. In the event that we cannot adjust the color of an icon, be consistent with your color decision throughout your entire choice tile set, rather than have some icons change colors and others not. The strong recommendation is to change the icon color with the selected states unless otherwise impossible.
```

Comment:
Who is "we" in this context?

Reply:
I'm unsure what this sentence is getting at, but lets try - "In the event the color of an icon cannot be changed, be consistent with your color decision throughout the entire choice tile set, rather than have some icons change colors and others not."